src/main/java/opendcs/dao/EnumSqlDao.java

```diff
@@ -87,7 +87,7 @@ public DbEnum getEnum(String enumName)
 				return ret;
 			
 			int dbVer = db.getDecodesDatabaseVersion();
-			String q = "SELECT " + getEnumColumns(dbVer) + " FROM Enum";
```

Comment:
oh, this wasn't actually needed, just had to figure out how to set the Calcite shema to not be case sensitive.